Formula/pipx.rb

```diff
@@ -0,0 +1,28 @@
+class Pipx < Formula
+  include Language::Python::Virtualenv
+
+  desc "Execute binaries from Python packages in isolated environments"
+  homepage "https://github.com/pipxproject/pipx"
+  url "https://files.pythonhosted.org/packages/b5/42/bdf9b6cc0af79222f69776d931ef79235946095b16594bac1f6bd81f435f/pipx-0.13.0.1.tar.gz"
+  sha256 "5134955d0d0595ca8040ecfa4215f2ddb84d34569c118ab2cde84b1d2b240b42"
+
+  depends_on "python"
+
+  def install
+    xy = Language::Python.major_minor_version "python3"
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
+    system "python3", *Language::Python.setup_install_args(libexec)
+    bin.install Dir["#{libexec}/bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  def caveats; <<~EOS
+    Run this command to ensure ~/.local/bin is on your PATH:
+      pipx ensurepath
+  EOS
+  end
```

Comment:
Also, I think the caveats block should go too; this advice is not specific to homebrew installation of pipx, AFAICT. Caveats are for homebrew specific stuff.

Reply:
@zbeekman I think this is actually fairly specific to `pipx` since adding this to the PATH is not necessarily required for normal python usage. Many people never use `pip install --user xyz` and thus they never need to have a path such as `~/.local/bin` in their PATH.

For `pipx` in particular, it will simply not work without this command being run. It's the first recommended step here in the installation instructions: https://github.com/pipxproject/pipx#install-pipx

What do you think?

Reply:
@Homebrew/maintainers: thoughts on this caveats block? I think it could go either way...

Reply:
Agreed on removing the `caveat` because it’s not HB-specific. That’s the approach we take on casks, but I don’t know how prevalent that is on formulae.

Reply:
Agreed: we don't want `caveats` to turn into general documentation.

Reply:
Sounds good- will remove.